app/requests_handlers.py

```diff
@@ -612,12 +628,29 @@ async def post(self, request):
                 else:
                     fulfilment_code_array.append(available_fulfilments[0]['fulfilmentCode'])
 
+                try:
+                    room_number = attributes['roomNumber']
+                except KeyError:
+                    room_number = None
+
+                if room_number:
+                    if len(attributes['last_name']) < last_name_char_limit:
```

Comment:
Just wondering about the case where the current last_name is just below the last_name limit (like last_name say 47 chars long and would be say limit+3 long once the room number is added).
Should this code not be comparing the length what last_name + room_number would be against the limit.
The if statement its self seems odd. It implies that it's ok to have a last_name which is longer than its limit :/